source/Tipster.cpp

```diff
@@ -1,9 +1,12 @@
 /*
  * Copyright 2015 Vale Tolpegin <valetolpegin@gmail.com>
+ * Copyright 2016 Ayush Agarwal <ayush94582@yahoo.com>
  * All rights reserved. Distributed under the terms of the MIT license.
  */
 #include "Tipster.h"
+
 #include <iostream>
```

Comment:
Sorry, I missed that before: <iostream> fits better farther down before <stdio.h>